### PR TITLE
Support command substitution

### LINKED ISSUE #63: Support Command Substitution
Currently, we support **Variable Substitution**. I think it could be great to support **command substitution**, too. Like the [dotenv from ruby](https://github.com/bkeepers/dotenv#command-substitution).

@jsl @sestrella @juanpaucar 
What do you think?

### PR DESCRIPTION
**Pull request by CristhianMotoche**
@juanpaucar Could you take a look, please? It closes #63 
TODO:
- [x] Update README
- [x] Update CHANGELOG

### CODE REVIEW

**Comment by CristhianMotoche** [comment]
I think it won't work for that syntax right now. I think I can add that capability if it is needed. 
Please, wait a little time and I'll be working on that. @juanpaucar

**Comment by juanpaucar** [comment]
@CristhianMotoche I think that it's no necessary to have support for that legacy syntax

**Comment by CristhianMotoche** [comment]
I think backtick are not legacy. Accordingly to the [Shell Command Language](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03) @juanpaucar 
However, we can add them when someone makes the request. WDYT?

**Comment by CristhianMotoche** [comment]
@juanpaucar Could you take a look, please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ## MASTER
+## Dotenv 0.5.1.2
```
Comment (juanpaucar) [inline CHANGELOG.md]: `5.1.0`*

**Pull request merged**